webui/src/lib/components/repository/refDropdown.jsx

```diff
@@ -221,7 +221,7 @@ const RefDropdown = ({ repo, selected, selectRef, onCancel, variant="light", pre
     const popover = (
         <Overlay target={target.current} show={show} placement="bottom" rootClose={true} onHide={() => setShow(false)}>
             <Popover className="ref-popover">
-                <Popover.Body>
+                <Popover.Content>
```

Comment:
Popover in v2.7.0 (which is the one we're now using) has a `Body` property instead of a `Content` one. 

see: https://github.com/react-bootstrap/react-bootstrap/blob/v2.7.0/src/Popover.tsx#L131

I'm guessing your npm dependencies are out of date (`npm install` or maybe ever delete your existing `node_modules`)

Reply:
additionally, make sure you re-generate the UI (`make gen-ui`)

Reply:
Yeah it worked after deleting `node_modules`, `npm install` by itself wasn't enough. Closing, sorry for the noise..